.pipelines/templates/e2e-template.yaml

```diff
@@ -0,0 +1,52 @@
+jobs:
+  - job: E2E_Tests
+    pool:
+      name: nodesigtest-pool
+    timeoutInMinutes: 90
+    displayName: Run AgentBaker E2E
+    variables:
+      - group: ab-e2e
+    steps:
+      - bash: |
+          az login --identity
+          az account set -s $(AZURE_SUBSCRIPTION_ID)
+        displayName: az login
+      - bash: bash .pipelines/scripts/setup_go.sh
+        displayName: setup go
+      - bash: |
+          export PATH="/usr/local/go/bin:$PATH"
+          go version
+          
+          echo "VHD_BUILD_ID=$VHD_BUILD_ID"
+          
+          if [ -n "${VHD_BUILD_ID}" ]; then
+            export SIG_VERSION_TAG_NAME=buildId
+            export SIG_VERSION_TAG_VALUE=$VHD_BUILD_ID
+            export IGNORE_SCENARIOS_WITH_MISSING_VHD=true
+          else
+            echo "Build.BuildId is not specified. Running default e2e tests."
+          fi
+          
+          cd e2e
+          go test -parallel 100 -timeout 90m -v -run Test_All ./
```

Comment:
could the parallel and timeout variable be added to the ab-e2e group ?

Reply:
Currently it contains only credentials. I think @cameronmeissner wants to delete them in a favour of managed identity

Reply:
I tried to make it configurable. But it's not trivial and the feedback loop of 1h+ is killing me.

Reply:
that's fair.  we can live with them hardcoded and turn them configurable if we end up needing to change them.